cmd/sharded-test-server/frontproxy.go

```diff
@@ -132,7 +132,7 @@ func startFrontProxy(
 		fmt.Sprintf("--root-directory=%s", filepath.Join(workDirPath, ".kcp-front-proxy")),
 		fmt.Sprintf("--root-kubeconfig=%s", filepath.Join(workDirPath, ".kcp/root.kubeconfig")),
 		fmt.Sprintf("--shards-kubeconfig=%s", filepath.Join(workDirPath, ".kcp-front-proxy/shards.kubeconfig")),
-		fmt.Sprintf("--client-ca-file=%s", filepath.Join(workDirPath, ".kcp/client-ca.crt")),
+		fmt.Sprintf("--client-ca-file=%s", filepath.Join(workDirPath, "test/e2e/framework/client-ca.crt")),
```

Comment:
I guess this is temporary, and we'll generate an extra CA cert like is [already done elsewhere](https://github.com/kcp-dev/kcp/blob/main/cmd/sharded-test-server/main.go#L69..L100) e.g we can create a new one in the [shard setup](https://github.com/kcp-dev/kcp/blob/main/cmd/sharded-test-server/shard.go) and rename the resulting files?

Reply:
yeah, this is temporary to see if we have other cascading e2e failures and it seems we do